eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/command/TeleportToRandomPlayerCommand.java

```diff
@@ -0,0 +1,56 @@
+package com.eternalcode.core.feature.teleport.command;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import com.eternalcode.core.notice.NoticeService;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+@Route(name = "teleport-to-random-player", aliases = { "tprp" })
+@Permission("eternalcore.tprp")
+public class TeleportToRandomPlayerCommand {
+
+    private static final Random RANDOM = new Random();
+
+    private final Server server;
+    private final PluginConfiguration pluginConfiguration;
+    private final NoticeService noticeService;
+
+    public TeleportToRandomPlayerCommand(Server server, PluginConfiguration pluginConfiguration, NoticeService noticeService) {
+        this.server = server;
+        this.pluginConfiguration = pluginConfiguration;
+        this.noticeService = noticeService;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Teleport to random player on server")
+    void execute(Player player) {
+        List<Player> applicablePlayers = this.server.getOnlinePlayers().stream()
+            .filter(players -> this.pluginConfiguration.teleport.includeOpPlayersInRandomTeleport || !players.isOp())
```

Comment:
```suggestion
            .filter(player -> this.pluginConfiguration.teleport.includeOpPlayersInRandomTeleport || !player.isOp())
```

Reply:
`player` is used to handle command sender.

Reply:
So maybe selectedPlayer?